src/devices/UFireIse/UFireIse.cs

```diff
@@ -0,0 +1,391 @@
+                SendCommand((byte)Command.ISE_CALIBRATE_SINGLE);
+
+                DelayHelper.DelayMilliseconds(IseMvMeasureTime, allowThreadYield: true);
+            }
+        }
+
+        /// <summary>
+        /// Calibrates the dual-point values for the low reading, in mV, and saves them in the devices's EEPROM.
+        /// </summary>
+        /// <param name="solution">Value. Range: -1024 mV to 1024 mV</param>
+        public void CalibrateProbeLow(ElectricPotential solution)
+        {
+            if (solution.Millivolts >= -1024 && solution.Millivolts <= 1024)
+            {
+                WriteRegister((byte)Register.ISE_SOLUTION_REGISTER, Convert.ToSingle(solution.Millivolts));
+                SendCommand((byte)Command.ISE_CALIBRATE_LOW);
+
+                DelayHelper.DelayMilliseconds(IseMvMeasureTime, allowThreadYield: true);
+            }
+        }
+
+        /// <summary>
+        /// Calibrates the dual-point values for the high reading, in mV, and saves them in the devices's EEPROM.
+        /// </summary>
+        /// <param name="solution">Value. Range: -1024 mV to 1024 mV</param>
+        public void CalibrateProbeHigh(ElectricPotential solution)
+        {
+            if (solution.Millivolts >= -1024 && solution.Millivolts <= 1024)
+            {
+                WriteRegister((byte)Register.ISE_SOLUTION_REGISTER, Convert.ToSingle(solution.Millivolts));
+                SendCommand((byte)Command.ISE_CALIBRATE_HIGH);
+
+                DelayHelper.DelayMilliseconds(IseMvMeasureTime, allowThreadYield: true);
+            }
+        }
+
+        /// <summary>
+        /// Returns the firmware version of the device.
+        /// </summary>
+        /// <returns>Firmware version</returns>
+        public Version GetVersion()
+        {
+            return new Version(ReadRegister((byte)Register.ISE_VERSION_REGISTER).ToString());
+        }
+
+        /// <summary>
+        /// Dual point uses two measures for low and high points. It needs the measured value (reading value) and the known value (reference value). Calling calibrateProbeLow saves both the reading and reference value.
+        /// When there are high and low calibration points, the device will automatically use them to adjust readings.To disable dual-point adjustment, call reset to remove all calibration data.
+        /// </summary>
+        /// <returns>The known value (reference value) for calibrate the high value</returns>
+        public ElectricPotential GetCalibrateHighReference()
+        {
+            return new ElectricPotential(ReadRegister((byte)Register.ISE_CALIBRATE_REFHIGH_REGISTER), UnitsNet.Units.ElectricPotentialUnit.Millivolt);
+        }
+
+        /// <summary>
+        /// Dual point uses two measures for low and high points. It needs the measured value (reading value) and the known value (reference value). Calling calibrateProbeLow saves both the reading and reference value.
```

Comment:
```suggestion
        /// Dual point uses two measures for low and high points. It needs the measured value (reading value) and the known value (reference value). Calling CalibrateProbeLow saves both the reading and reference value.
```

please ensure this name is correct